SPECS/golang/golang-1.15.spec

```diff
@@ -32,18 +32,18 @@ Go is an open source programming language that makes it easy to build simple, re
 %prep
 # Setup go 1.4 bootstrap source
 tar xf %{SOURCE1} --no-same-owner
-patch -Np1 --ignore-whitespace < /usr/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
+patch -Np1 --ignore-whitespace < %{_prefix}/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
 mv -v go go-bootstrap
 
 %setup -q -n go
 
 %build
 # Build go 1.4 bootstrap
-pushd /usr/src/mariner/BUILD/go-bootstrap/src
+pushd %{_prefix}/src/mariner/BUILD/go-bootstrap/src
 CGO_ENABLED=0 ./make.bash
 popd
-mv -v /usr/src/mariner/BUILD/go-bootstrap /usr/lib/golang
-export GOROOT=/usr/lib/golang
+mv -v %{_prefix}/src/mariner/BUILD/go-bootstrap %{_libdir}/golang
```

Comment:
I would revert the use of the `%{_prefix}` macro in this case and above.

Why: the path macros are related to the system where the package gets installed, not the build environment, like in this case. We're installing our `golang` package for the build environment, which uses our path macros to define where the binaries get installed to, so the path on the right should always be fine (changes to the macro will propagate to the package). I wouldn't be so sure about that when it comes to the path on the left hand side - it's defined by `rpm` and I think we don't control that.